parsec/backend/postgresql/invite.py

```diff
@@ -462,9 +462,15 @@ async def _do_new_user_invitation(
 
 
 class PGInviteComponent(BaseInviteComponent):
-    def __init__(self, dbh: PGHandler, *args, **kwargs):
+    def __init__(self, dbh: PGHandler, user_component: PGUserComponent, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.dbh = dbh
+        self.user_component = user_component
+
+    async def user_already_member(self, organization_id: OrganizationID, email: str) -> bool:
+        async with self.dbh.pool.acquire() as conn, conn.transaction():
+            r = await self.user_component.retrieve_human(organization_id, email)
```

Comment:
note here we acquire a connection and create a transaction... then call `self.user_component.retrieve_human` that itself acquire a connection and create a transaction !

We should instead use the `query_retrieve_human` function that takes a conn object and is expected to be executed in an existing transaction context if needed.